### ISSUE TITLE
Wrong capitalisation in "Guests on ride x" and "Guests queuing for ride x"

### ISSUE THREAD

**Issue by SpicySpaceman** [OpenRCT2/OpenRCT2#1514]
![2015-06-28 21_21_33-openrct2](https://cloud.githubusercontent.com/assets/10995998/8397915/3ce8d524-1ddc-11e5-9a44-897b7fea8926.png)
Should be "Guests on Jetski 1"

![2015-06-28 21_21_45-openrct2](https://cloud.githubusercontent.com/assets/10995998/8397916/3d8fe8fa-1ddc-11e5-9dd0-16f8687398df.png)
Should be "Guests queuing for Jetski 1"

Caused by combining two strings, four new strings would solve this issue.

`Guests on {STRINGID}`
`Guests in {STRINGID}`
`Guests at {STRINGID}`
`Guests queuing for {STRINGID}`

**Comment by Gymnasiast** [comment]
It would require duplication of _all_ guest actions, not just those concerning rides. Also, English has looser capitalisation rules. In any way, it's jumping into conclusions that it's wrong.

**Comment by ThomasdenH** [comment]
You could easily do the same thing for English: Guests with status: {}, but I definitely find it less elegant.

**Comment by danidoedel** [comment]
Damn, thought that would be easy for you guys :P

**Comment by ThomasdenH** [comment]
If it is just the capital and not the grammar, that could noted easily be corrected.

**Comment by groenroos** [comment]
> If it is just the capital and not the grammar, that could relatively easily be corrected.

While in English it's only the capital letter, in other languages also the grammar breaks down, as noted above and in #15453.

In my opinion, the solution is for all of the guest status strings `STR_1431` thru `STR_1448` to be duplicated, with the new set of strings used exclusively for interpolating in `STR_1820` (`Guests {STRINGID}`).

**Comment by 733737** [comment]
I'd like to work on a pr to implement these changes if @Gymnasiast isn't opposed to it.

**Comment by Gymnasiast** [comment]
Yes, go ahead. You should only update en-gb, by the way, the rest (including en-us) is handled on the Localisation repo
